pandas/_libs/parsers.pyx

```diff
@@ -1479,6 +1449,7 @@ cdef _string_box_factorize(parser_t *parser, int64_t col,
 
     return result, na_count
 
+
```

Comment:
are we not linting these .pyx for the 2 blank lines between functions?

Reply:
flake8 chokes on a SyntaxError as soon as it hits a `cimport`